01. 기본문제 풀이/02. 삼각형 판별하기/김보경/index.js

```diff
@@ -1,7 +1,13 @@
 function solution(a, b, c) {
-  let answer = 'YES';
+  let answer = "YES";
 
-  return answer;
+  let arr = [a, b, c];
+  let maxValeu = Math.max(...arr);
+  let result = arr.reduce((acc, el) => {
+    return el !== maxValeu ? acc + el : acc;
+  });
+
+  return result > maxValeu ? answer : "NO";
```

Comment:
보경님도 기존의 코드를 변경하는 것을 안 좋아하시나요?
answer를 사용한다면 answer에 값을 정해주고 answer를 리턴하는 것이 낫지 않을까라는 생각이 듭니다~

- answer 없이
```suggestion
  return result > maxValeu ? "YES": "NO";
```
- answer 사용
```suggestion
  answer = result > maxValue ? "YES" : "NO";
  return answer;
```